repl/src/main/resources/fake_shell.py

```diff
@@ -368,6 +369,11 @@ def main():
     sys.stderr = fake_stderr
 
     try:
+
+        if os.environ["LIVY_TEST"] == "false":
```

Comment:
This will cause an error if the variable is not set.

    >>> import os
    >>> os.environ['LIVY_TEST']
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/usr/lib/python2.7/UserDict.py", line 23, in __getitem__
        raise KeyError(key)
     KeyError: 'LIVY_TEST'

Instead:

    if os.environ.get('LIVY_TEST') == "true": ...